src/OrbitBase/include/OrbitBase/ProcessId.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
```

Comment:
I'm not a fan of the name of this file, it sounds like a type rather than describing what these functions are for.

From the include guards I see you had gone for "LinuxProc", maybe that's also not ideal but I liked it more.

What about "ListProcesses.h"? Or "ListProcessIds.h"?

Reply:
Yes, the name is not great. I wanted to change away from "LinuxProc" because I don't want "Linux" in the name (the name should describe what is done not the platform, also this would result in LinuxProcLinux.cpp which is weird).
"ListProcessIds.h" sounds ok-ish to me. 

I looked around in OrbitBase; there is already a file "ThreadUtils.h". I now think moving the stuff there is the best solution. Agree?

Reply:
I kind of like having them separate, with these free-standing function in `OrbitBase` it's nicer to have small clearly-separate files (matches better the include what you use philosophy).

As for the name, if you choose to rename the functions to something other than "Get" (but for me they're also fine as they are), then the name of the file could match that (e.g., with "Fetch" this could be called "FetchProcessIds.h").

Reply:
GetProcessIds.{h,cpp} it is.